addons/udes_stock/models/product_template.py

```diff
@@ -2,9 +2,27 @@
 
 from odoo import fields, models
 
+
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
+    def _domain_product_category(self, category):
+        """Domain for product categories, not including category itself"""
+        Category = self.env['product.category']
+        subcategories = Category.search([('id', 'child_of', category.id)])
+        return [('id', 'in', subcategories.ids),
+                ('id', '!=', category.id)]
```

Comment:
Can we not just return a domain which uses child_of directly, rather than building an explicit list of ids?

Reply:
yes definetly.